test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/DicomServerTestCategories.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Dicom.Web.Tests.E2E.Common;
+
+[Flags]
+public enum DicomTestServerCategory : byte
+{
+    None,
```

Comment:
Could you add comment for each, so it will be easily understood later

Reply:
Done.